configs/milvus.yaml

```diff
@@ -48,6 +48,10 @@ pulsar:
   port: 6650 # Port of pulsar
   maxMessageSize: 5242880 # 5 * 1024 * 1024 Bytes, Maximum size of each message in pulsar.
 
+kafka:
+  address: localhost
+  port: 9092
+
```

Comment:
should be like this
```go
kafka:
  brokers:
    - localhost:9092
```
It can connect kafka cluster

Reply:
normally this is a kubernetes service address. I would prefer use address instead of brokers.

Reply:
In kubernetes, it should be headless service .

Reply:
why use headless service. If you scale out the kafka cluster, should we change the `brokers` config?

Reply:
It's a matter of choice. If you use service, he will only access one randomly. For example , if use redis in kubernetes , must use headless services. Just like you said, scale the pod  is getting really stupid.